presto-hive/src/main/java/com/facebook/presto/hive/HiveTableLayoutHandle.java

```diff
@@ -66,7 +67,8 @@ public HiveTableLayoutHandle(
             @JsonProperty("partitionColumnPredicate") TupleDomain<ColumnHandle> partitionColumnPredicate,
             @JsonProperty("bucketHandle") Optional<HiveBucketHandle> bucketHandle,
             @JsonProperty("bucketFilter") Optional<HiveBucketFilter> bucketFilter,
-            @JsonProperty("layoutString") String layoutString)
+            @JsonProperty("layoutString") String layoutString,
+            @JsonProperty("pushdownFilterEnabled") boolean pushdownFilterEnabled)
```

Comment:
pushdownFilterEnabled argument belongs before layoutString; layoutString should be the last argument